org.feichtmeier.Musicpod.json

```diff
@@ -0,0 +1,230 @@
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "squashfs-tools",
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "XZ_SUPPORT=1 LZO_SUPPORT=0 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "git",
+                                   "url": "https://github.com/plougher/squashfs-tools.git",
+                                   "tag": "4.6.1",
+                                   "commit": "d8cb82d9840330f9344ec37b992595b5d7b44184",
+                                   "x-checker-data": {
+                                        "type": "git",
+                                        "tag-pattern": "^([\\d.]+)$"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "musicpod",
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "unsquashfs -quiet -no-progress *.snap",
+                              "cp -r squashfs-root/bin /app",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.desktop /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "desktop-file-edit --set-key=Icon --set-value='org.feichtmeier.Musicpod' /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.png /app/share/icons/hicolor/256x256/apps/org.feichtmeier.Musicpod.png",
+                              "install -Dm644 org.feichtmeier.Musicpod.metainfo.xml /app/share/metainfo/org.feichtmeier.Musicpod.metainfo.xml"
+                         ],
+                         "cleanup": [
+                              "/include",
+                              "/lib/pkgconfig",
+                              "/share/man"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "file",
+                                   "path": "org.feichtmeier.Musicpod.metainfo.xml"
+                              },
+                              {
+                                   "type": "file",
+                                   "filename": "musicpod.snap",
+                                   "only-arches": [
+                                        "x86_64"
+                                   ],
+                                   "url": "https://api.snapcraft.io/api/v1/snaps/download/sxdOSGRbzsWhPmGrkYYAs68rzieTrIiv_658.snap",
```

Comment:
why not building from source instead?

Reply:
Hi thanks for the review!

Since I have build a CI/CD chain for the snap and the tooling for flutter snaps was something that I could manage on my own, this solution is easier for me to maintain since my knowledge in the flatpak tech is very limited. 
I am not sure if this is a reason for you but that is at least my explanation :)